src/module.ts

```diff
@@ -72,3 +79,38 @@ export default defineNuxtModule<ModuleOptions>({
     configureVite(CONFIG_KEY, nuxt, ctx)
   },
 })
+
+export interface ModuleHooks {
+  'vuetify:registerModule': (registerModule: (config: InlineModuleOptions) => void) => HookResult
+}
+
+export interface RuntimeModuleHooks {
+  'vuetify:configuration': (options: {
+    isDev: boolean
+    vuetifyOptions: VuetifyOptions
+  }) => HookResult
+  'vuetify:before-create': (options: {
+    isDev: boolean
+    vuetifyOptions: VuetifyOptions
+  }) => HookResult
+  'vuetify:ready': (vuetify: ReturnType<typeof createVuetify>) => HookResult
+  'vuetify:ssr-client-hints': (options: {
+    vuetifyOptions: VuetifyOptions
+    ssrClientHints: SSRClientHints
+    ssrClientHintsConfiguration: SSRClientHintsConfiguration
+  }) => HookResult
+}
+
+declare module '#app/nuxt' {
```

Comment:
should be `#app`: https://github.com/nuxt/nuxt/issues/24193#issuecomment-1809959962